### ISSUE TITLE
Write and test code to create training inputs

### ISSUE THREAD

**Issue by aamster** [AllenInstitute/DeepCell#31]
Write a script that does the following:
1. Queries cell_labeling_app db to pull all ROIs that were labeled `LABELERS_REQUIRED_PER_REGION` times (this should be an input to the script). Parses labels in `user_labels.labels` field to produce a label for each ROI. Whether we are using `consensus` or `majority` should be an input to the script. 
2. For each ROI, using [this class](https://github.com/AllenInstitute/ophys_etl_pipelines/blob/main/src/ophys_etl/modules/roi_cell_classifier/compute_classifier_artifacts.py#L60), generate a model input
3. Create a list of [ModelInput](https://github.com/AllenInstitute/DeepCell/blob/0ca501eefeac27162b1db4d8371c9a6d996fbc40/src/deepcell/datasets/model_input.py#L8) for each ROI using the paths created in (2). Create a list of [`ExperimentMetadata`](https://github.com/AllenInstitute/DeepCell/blob/bb0df9f8c90959c8ce0411f5f942ab55f21f1650/src/deepcell/datasets/exp_metadata.py#L9)  **only for experiments in the list of `ModelInput`**. 
4. Call [`DataSplitter.get_train_test_split`](https://github.com/AllenInstitute/DeepCell/blob/bb0df9f8c90959c8ce0411f5f942ab55f21f1650/src/deepcell/data_splitter.py#L116) on the list of `ModelInput` and `ExperimentMetadata` from (3) to produce a train and test list of `ModelInput`.
5. Instantiate `VisualBehaviorExtendedDataset` class to pull all previously existing inputs. The `dataset` property contains `List[ModelInput]`. Filter this field by `project_name` to include only `add_vasculature` and `large_processes` as this training data was found to help the model. Append the list of _train_ inputs from (4) with this list.
6. Dump the train and test set to disk using the [`ModelInputSchema`](https://github.com/AllenInstitute/DeepCell/blob/0ca501eefeac27162b1db4d8371c9a6d996fbc40/src/deepcell/cli/schemas/data.py#L12)as `train.json` and `test.json`

Write down in this ticket the location of the files from (6)

**Comment by morriscb** [comment]
Finished code review and checking to see that the code does indeed work with the input data from the cell labeling app db and input experiment metadata. Looks like we are ready for the data to come in once it's available. Here's the file for the metadata: [Experiment metadata](https://alleninstitute-my.sharepoint.com/:u:/g/personal/chris_morrison_alleninstitute_org/EaGaj0f_smNBoLzzrdgNi-gBf63lYzzq4B6h2mClBx9GEg?e=Tt2BfB)

**Comment by morriscb** [comment]
I dumped an initial set of ROI data (sans artifacts) to json. Here are the 3 files:
[train_rois](https://alleninstitute-my.sharepoint.com/:u:/g/personal/chris_morrison_alleninstitute_org/EYVngndcp1hNuype97GMga8BTBJpgQwk-vrxTLzPueboHQ?e=YHqEjr): The set of rois for training from the SSF dataset. We are going to reproduce these at a later date since labeling is still ongoing.
[test_rois](https://alleninstitute-my.sharepoint.com/:u:/g/personal/chris_morrison_alleninstitute_org/EVyiYYYtnwFPt4abu9OkggsBeA1HdG-9CUv4RAaQ-97qKA?e=Ht2CUF): Test set of ROIs from the SSF dataset. Again these are recreated after the labeling effort is finished.
[train_rois_vb_extended](https://alleninstitute-my.sharepoint.com/:u:/g/personal/chris_morrison_alleninstitute_org/EZYNtHZ1-5RKsKJ02mS3cLUB1aKe06J8vuS1PDTkGZbpIg?e=2e6jYj) Set of rois from VisualBehaviorExtendedDataset exluding, `ophys-experts-go-big-or-go-home`, `ophys-experts-slc-oct-2020_ophys-experts-go-big-or-go-home`, `ophys-expert-danielsf-additions`, datasets. Not sure if this results in the desired ones as mentioned above but recreating these jsons is easy enough when the labeling job is complete.

**Comment by morriscb** [comment]
One feature to discuss later is if we want to address the "these experiments are always reserved for the test set" now or in the future once we see how many regions we have labeled. If all 240 get 3 labels, this is likely a non-issue.
